react/features/gifs/functions.js

```diff
@@ -81,6 +81,10 @@ export function isGifEnabled(state) {
     const { disableThirdPartyRequests } = state['features/base/config'];
     const { giphy } = state['features/base/config'];
 
+    if (navigator.product === 'ReactNative' && window.JITSI_MEET_LITE_SDK) {
```

Comment:
Is this thing ok on the window?

Reply:
Yeah, check custom.js, where I attach it to global. Using window instead of global would've been the same, but I opted for this for consistency with other polyfills.

Reply:
I'll follow up with some polyfills cleanup.